backend/oasst_backend/prompt_repository.py

```diff
@@ -312,10 +317,9 @@ def insert_toxicity(self, message_id: UUID, model: str, score: float, label: str
 
         message_toxicity = MessageToxicity(message_id=message_id, model=model, score=score, label=label)
         self.db.add(message_toxicity)
-        self.db.commit()
-        self.db.refresh(message_toxicity)
```

Comment:
we could first keep all existing refresh() code .. just replacing `db.commit()` by `db.flush()` ...

Reply:
if the intention is to get the database-side value for all the attributes of Message then this is achieved by the FLUSH in the decorator. It turns the lazy loading to eager loading. Message will contain all the values before it is returned to the caller. The refresh is not required here. let me know your opnion